static/src/javascripts/projects/commercial/modules/header-bidding/prebid/bid-config.spec.js

```diff
@@ -524,3 +525,79 @@ describe('triplelift adapter', () => {
         });
     });
 });
+
+describe('getXaxisPlacementId', () => {
+    beforeEach(() => {
+        resetConfig();
+        getBreakpointKey.mockReturnValue('D');
+    });
+
+    afterEach(() => {
+        jest.resetAllMocks();
+    });
+
+    const generateTestIds = () => {
+        const prebidSizes = [
+            [[300, 250]],
+            [[300, 600]],
+            [[970, 250]],
+            [[728, 90]],
+            [[1, 2]],
+        ];
+        return prebidSizes.map(getXaxisPlacementId);
+    };
+
+    test('should return -1 if no cases match', () => {
+        expect(getImprovePlacementId([[1, 2]])).toBe(-1);
+    });
+
+    test('should return the expected values for desktop device', () => {
+        getBreakpointKey.mockReturnValue('D');
+        containsMpuOrDmpu.mockReturnValueOnce(true);
+        containsMpuOrDmpu.mockReturnValueOnce(true);
+        containsMpuOrDmpu.mockReturnValue(false);
```

Comment:
These mocks can be [chained][], and maybe extracted to the `beforeEach`?

```suggestion
        containsMpuOrDmpu
            .mockReturnValueOnce(true);
            .mockReturnValueOnce(true);
            .mockReturnValue(false);
```

[chained]: https://jestjs.io/docs/en/mock-function-api#mockfnmockreturnvalueoncevalue